Kwenta/kwenta
# Added XMR to future markets

## Description
Add a new market: XMR-PERP

## Related issue
#1356 

## Motivation and Context


## How Has This Been Tested?
1. Open the market page and switch to XMR market
2. Open the dashboard page and see the XMR on the list

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/188828644-de2bde08-d8fe-45f3-b3c8-e2c1ce373012.png)
![image](https://user-images.githubusercontent.com/4819006/188828887-333a3c48-eb71-4be0-bd2c-6f03d0a7e34b.png)
